.metals/metals.lock.db

```diff
@@ -0,0 +1,6 @@
+#FileLock
+#Thu Jul 08 09:49:44 BST 2021
+hostName=localhost
```

Comment:
We don't usually version these files and I suspect they should be included in the `.gitignore`. It should probably be removed from this PR.

Reply:
Ta, just done that.